### DISCUSSION TITLE
Mark all as read on android app

### DISCUSSION THREAD

**Discussion by WillTheDeveloper** [community/community#6518] [Mobile] [answered]
Unless I have missed it, I was unable to find a button that allows me to mark all notifications as read on the GitHub android app and I think it would be really important to have if you have like 50 notifications all the time. Need a button adding to the top right or something to mark them all as read. Would be super handy.

**Comment by amrehman** [comment, marked as answer]
Hello there! 
Sorry to hear that you are having some trouble. This feature is available let me try to walk you through it.

Perhaps its not that intuitive, here is what you have to do:
- Long tap on one of the notifications.
- Then tap on the vertical ellipsis button on the navigation bar, tap select all and then tap the Mark as Read button. 

I will create a follow up for the team to make this more discoverable.

Thanks for the feedback!

**Reply by WillTheDeveloper** [reply]
Okay, Thanks for that 😄
